.travis.yml

```diff
@@ -0,0 +1,51 @@
+dist: xenial
```

Comment:
This should be removed when it is ready

Reply:
I think it should be able to be removed right now because the CI is already triggered by gitlab.

Reply:
Yeah, this is just `CI` for the `goerli fork`

Reply:
removing it once we have at least one approval here